change/@internal-react-components-97ccb55b-fe63-4cc1-8003-18bf04a13072.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "exposing styling through composite",
```

Comment:
nit: for changelog entries try making entries both specific to the package, and clear what has changed to a user who may read this. E.g. for react-components package this should probably be `Expose chat message styling through MessageThread interface`.

Reply:
Would I create a new rush changelog or is it safe to just rename these entries?